contracts/protocols/mainnet/euler/helpers.sol

```diff
@@ -0,0 +1,235 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.6;
+pragma abicoder v2;
+import "./interface.sol";
+
+contract EulerHelper {
+    address internal constant EULER_MAINNET = 0x27182842E098f60e3D576794A5bFFb0777E025d3;
+
+    IEulerMarkets internal constant markets = IEulerMarkets(0x3520d5a913427E6F0D6A83E07ccD4A4da316e4d3);
+
+    IEulerGeneralView internal constant eulerView = IEulerGeneralView(0xACC25c4d40651676FEEd43a3467F3169e3E68e42);
+
+    struct SubAccount {
+        uint256 id;
+        address subAccountAddress;
+    }
+
+    struct Position {
+        uint256 id;
+        address subAccountAddress;
+        AccountStatus accountStatus;
+        MarketsInfoSubacc[] marketsInfoSubAcc;
+    }
```

Comment:
We need user health score as well

Reply:
I think for each SubAccount health factor will be different.

Reply:
Oh yes! I've missed the health factor. Yes, it will be different for each subAccount.

Reply:
Done!